### DISCUSSION TITLE
Assign Audio Feed to Channel (5 - 7 point speakers)

### DISCUSSION THREAD

**Discussion by DxBang** [f1multiviewer/issue-tracker#488] [Feature requests] [answered]
I think it could be awesome to be able to assign a specific audio stream to a specific set of audio channels.
e.g.

- Assign Audio to Channel:
  - Default
  - 2.1 Stereo Speakers + Sub
  - Center Front
  - Side Stereo Speakers
  - Rear Stereo Speakers
  - Front Left Speaker
  - Front Right Speaker
  - Side Left Speaker
  - Side Right Speaker
  - Rear Left Speaker
  - Rear Right Speaker

So it is possible to set the Main Feed to Front Speakers and have the drivers at Side Left, Rear Left, Side Right, Rear Right or where ever you wish to hear those teamradio conversations from.

So as a simple stereo... main feed on your left channel, and your selected teamradio feeds on the right channel...

**Comment by ExtremTechniker** [comment]
Nice idea!
Than it would be possible to use it with e.g. an audio router to extend and mix all the feed independently.

**Comment by f1multiviewer** [comment, marked as answer]
This is likely out of scope for us. What you can do instead is to set up a audio loopback device and do this through other software, and then for each stream, use the virtual audio output as the audio output device (available on right-click).
